### DISCUSSION TITLE
Add an Apply option for the GPU section

### DISCUSSION THREAD

**Discussion by Valour549** [seerge/g-helper#3211] [Ideas]
In Fans and Power, the CPU section and the Fans section have their own "apply" boxes.

I think it would make good sense to also have an apply box for the GPU section. It's a lil scary to instantly BSOD (which can happen if you are in Ultimate GPU mode) just because you've dragged the bars a little too far accidentally, particularly if limits have been raised in the config file.

**Comment by seerge** [comment]
@Valour549 hello, you won't get BSOD by dragging sliders in GPU section. If you are speaking about Core Clock / Memory Clock offsets - they affect only max performance state for Nvidia GPU, that is active only when you run a demanding game or application and not on desktop. If you overclock too much - game may crash (and close to desktop), and that means you need to set lower overclocks.
